HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -1746,6 +1746,10 @@ def self.apply_detailed_performance_data_for_var_speed_systems(hpxml_bldg)
   end
 
   def self.apply_hvac_control(hpxml_bldg, schedules_file, eri_version)
+    if hpxml_bldg.hvac_controls.size == 0 && hpxml_bldg.has_location(HPXML::LocationOtherHeatedSpace)
+      hpxml_bldg.hvac_controls.add(id: "HVACControl#{hpxml_bldg.hvac_controls.size + 1}")
+    end
```

Comment:
Note that any solution will have to address all the MF space types whose temperatures are based on the conditioned zone's temperatures, not just "other heated space".

That said, I don't think this is the right solution. Maybe it causes the simulation to run, but you end up with an "other heated space" that is the average of the conditioned zone temperature and outdoor zone -- except the conditioned zone is not conditioned. So it does not reasonably represent "other **heated** space" in any way.

My immediate reaction is that these space types should simply use default setpoints in the [EMS program](https://github.com/NREL/OpenStudio-HPXML/blob/0200043b5a9ceeb266837fed4e0fd56e2cdb60ad/HPXMLtoOpenStudio/measure.rb#L2786) (similar to what we do for [natural ventilation](https://github.com/NREL/OpenStudio-HPXML/blob/0200043b5a9ceeb266837fed4e0fd56e2cdb60ad/HPXMLtoOpenStudio/resources/airflow.rb#L428-L430)) rather than use a conditioned zone temperature sensor.

Reply:
I like Scott's instinct here. Using the setpoint rather than the space temp (which probably won't be that far off from outside if it's truly unconditioned, maybe a little higher from internal + solar gains) doesn't do a good job of representing a heated space.